app/views/teams/_admin_info_table.html.haml

```diff
@@ -5,9 +5,12 @@
     %table.table.table-bordered.table-striped
       %thead
         %tr
-          %th User ID
-          %th Name
-          %th Email
+          %th 
```

Comment:
There are a few spots where we use the style
`%th= t(...`
and then a few where we use the style
```
%th
  = t(...
```

For consistency could you make them all the first style?

Reply:
Fixed